app/assets/javascripts/discourse/widgets/search-menu.js.es6

```diff
@@ -97,6 +97,7 @@ export default createWidget('search-menu', {
     const results = [this.attach('search-term', { value: searchData.term, contextEnabled }),
                      this.attach('search-context', { contextEnabled })];
 
+    if (searchData.term)
```

Comment:
You need to update this to use braces. `if` statements without them are so error prone for refactoring later on. Additionally, I'm surprised this works without the additional check of `searchData.term.length > 0`